views/templates/edit.hbs

```diff
@@ -0,0 +1,19 @@
+<div id="edit_container">
+  <div id="edit_tooltip">Submit a post:</div>
+
+  {{#with featured}}
+  <form action="/gallery/{{id}}?_method=PUT" method="POST" id="edit_makePost">
+    {{!-- <div id="edit_inputContainer">
+      <input type="text" value="author" placeholder="Author" id="edit_author">
+      <input type="text" value="link" placeholder="URL" id="edit_link">
+    </div> --}}
+
+    <div id="edit_submitContainer">
+      <input type="text" name="author" value="author" placeholder="Author" id="edit_author">
```

Comment:
These input fields should be populated with the properties of gallery that you're trying to edit.